config/clusters/allen-swdb/common.values.yaml

```diff
@@ -0,0 +1,133 @@
+basehub:
+  nfs:
+    pv:
+      # from https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-nfs-mount-settings.html
+      mountOptions:
+        - rsize=1048576
+        - wsize=1048576
+        - timeo=600
+        - soft # We pick soft over hard, so NFS lockups don't lead to hung processes
+        - retrans=2
+        - noresvport
+      serverIP: fs-09e1be5c1c6f61028.efs.us-west-2.amazonaws.com
+      baseShareName: /
+  jupyterhub:
+    custom:
+      2i2c:
+        add_staff_user_ids_to_admin_users: true
+        add_staff_user_ids_of_type: "github"
+      homepage:
+        templateVars:
+          org:
+            name: Allen Institute Summer Workshop on the Dynamic Brain
+            logo_url: https://github.com/AllenInstitute/swdb_2022/blob/main/resources/cropped-SummerWorkshop_Header.png?raw=true
+            url: https://github.com/AllenInstitute/swdb_2022
+          designed_by:
+            name: 2i2c
+            url: https://2i2c.org
+          operated_by:
+            name: 2i2c
+            url: https://2i2c.org
+          funded_by:
+            name: Allen Institute
+            url: https://alleninstitute.org/
+    singleuser:
+      defaultUrl: /lab
+      initContainers:
+        # Need to explicitly fix ownership here, since EFS doesn't do anonuid
+        - name: volume-mount-ownership-fix
+          image: busybox
+          command:
+            [
+              "sh",
+              "-c",
+              "id && chown 1000:1000 /home/jovyan /home/jovyan/shared && ls -lhd /home/jovyan",
+            ]
+          securityContext:
+            runAsUser: 0
+          volumeMounts:
+            - name: home
+              mountPath: /home/jovyan
+              subPath: "{username}"
+            - name: home
+              mountPath: /home/jovyan/shared
+              subPath: _shared
```

Comment:
This part can go after https://github.com/2i2c-org/infrastructure/pull/1591 gets merged, right?

Reply:
@GeorgianaElena yep! I just merged that, will rebase and get rid of this!